### PR TITLE
support multiples of the same youtube atoms on the same page

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

Fixes a bug which prevented a youtube atom being enhanced if the same atom existed elsewhere on the page (because they shared the same ID).

## What is the value of this and can you measure success?

We can have an atom on the page twice.

## Does this affect other platforms - Amp, Apps, etc?

No

## Screenshots

N/A

## Tested in CODE?

No

@akash1810 @gidsg

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js [hunk 1]
```diff
@@ -177,27 +175,32 @@ define([
...
                 youtubePlayer.init(iframe, {
                     onPlayerReady: onPlayerReady.bind(null, atomId, overlay),
                     onPlayerStateChange: onPlayerStateChange.bind(null, atomId)
-                }, youtubeId);
+                }, iframe.id);
             });
         });
     }
 
+    function getTrackingId(atomId) {
```
Comment (akash1810) [inline static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js]: could we add a comment as to why this is needed?

**Review by akash1810 — APPROVED** [review]
could we add an inline comment, other than that 👍

**Comment by gidsg** [comment]
Am I right in understanding the fundamental issue is multiple videos having the same `iframe id`? If so perhaps we could solve this server side, say by appending a UUID to the iframe ID with a delimiting character? 

Happy for this to go out as-is though and we can think about this in a subsequent PR.

File: static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js (hunk 1 above)
Comment (GHaberis) [inline static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js]: Yep, I'll do that!

**Review by gidsg — APPROVED** [review]

**Comment by gidsg** [comment]
...actually I think a completely random string is not viable as it would dilute the cache. Back to the drawing board!

**Pull request merged**